tox.ini

```diff
@@ -4,17 +4,12 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26,py27,cover,lint
+envlist = py26,py27,lint
```

Comment:
`pytest --cov-min` is available only in yet unreleased 2.0 branch: https://github.com/schlamar/pytest-cov/commit/8f679db735ab99f66808e79d4e401996a9191ec0